kmeans.py

```diff
@@ -28,12 +28,25 @@ def add(self, p):
         self.members.append(p)
         return self.recalc_center()
     
-    def recalc_center()->float:
+    def recalc_center(self)->float:
         '''returns eclidean Distance, between the updated centroid to the previous one'''
-        pass
+        coords = () * self.center.dimention
+        for i in range(self.center.dimention):
+            sum = 0
+            for point in self.members:
+                sum += point.coord[i]
+            
+            coords = coords + (sum / len(self.members), )
```

Comment:
This leaves an empty spot after the last coordinate is calculated. 
Meaning we'll get:
`(x0, x1,..., xn,   )`
`_________________↑`
This does not show up when printing the coordinates but it takes memory for no reason

Reply:
when I do the change you've suggested it doesn't work, you want to try it ?